gdml/other/HERASouthHallSimple6ScintillatorPairs.gdml

```diff
@@ -0,0 +1,532 @@
+    <constant name="outsidegroundYSize" value="world_size_Y" />
+    <constant name="outsidegroundZSize" value="wall1ZSize + +roof1ZSize + 740" />
+    <constant name="insidegroundXSize" value="35500" />
+    <constant name="insidegroundYSize" value="46000" />
+    <constant name="insidegroundZSize" value="wall1ZSize + roof1ZSize+ 840" />
+    <position name="groundPosition" unit="mm" x="3500" y="0" z="-world_size_Z/2 + outsidegroundZSize/2"/>
+    <constant name="roofgroundXSize" value="16700" />
+    <constant name="roofgroundYSize" value="46000" />
+    <constant name="roofgroundZSize" value="740" />
+    <position name="roofgroundPosition" unit="mm" x="-5900" y="0" z="-world_size_Z/2 + 24200 + roof1ZSize + roofgroundZSize/2"/>
+    <constant name="blockgroundXSize" value="8500" />
+    <constant name="blockgroundYSize" value="20150" />
+    <constant name="blockgroundZSize" value="24700 + 740" />
+    <position name="blockgroundPosition" unit="mm" x="18000" y="-12925" z="-world_size_Z/2 + blockgroundZSize/2"/>
+    <constant name="miniroofXSize" value="8000" />
+    <constant name="miniroofYSize" value="24850" />
+    <constant name="miniroofZSize" value="200" />
+    <position name="miniroof1Position" unit="mm" x="16750" y="10075" z="-world_size_Z/2 + miniroofZSize/2 + 3000"/>
+    <position name="miniroof2Position" unit="mm" x="16750" y="10075" z="-world_size_Z/2 + miniroofZSize/2 + 6000"/>
+    <position name="miniroof3Position" unit="mm" x="16750" y="10075" z="-world_size_Z/2 + miniroofZSize/2 + 9000"/>
+    <position name="miniroof4Position" unit="mm" x="16750" y="10075" z="-world_size_Z/2 + miniroofZSize/2 + 12000"/>
+    <position name="miniroof5Position" unit="mm" x="16750" y="10075" z="-world_size_Z/2 + miniroofZSize/2 + 15000"/>
+    <position name="miniroof6Position" unit="mm" x="16750" y="10075" z="-world_size_Z/2 + miniroofZSize/2 + 18900"/>
+    <position name="miniroof7Position" unit="mm" x="16750" y="10075" z="-world_size_Z/2 + miniroofZSize/2 + 24240"/>
+    <position name="miniroof8Position" unit="mm" x="16750" y="10075" z="-world_size_Z/2 + miniroofZSize/2 + 27140"/>
+  </define>
+
+
+  <!-- Material definitions -->
+  <materials>
+    <material name="Scintillator" state="solid">
+      <T unit="K" value="293.15"/>
+      <D unit="g/cm3" value="1.18"/>
+      <fraction n="0.6" ref="C" />
+      <fraction n="0.32" ref="O" />
+      <fraction n="0.08" ref="H" />
+    </material>
+
+    <material name="Ground" state="solid">
+      <T unit="K" value="293.15"/>
+      <D unit="g/cm3" value="2"/>
+      <fraction n="0.4" ref="Si" />
+      <fraction n="0.3" ref="O" />
+      <fraction n="0.1" ref="Al" />
+      <fraction n="0.05" ref="Fe" />
+      <fraction n="0.03" ref="C" />
+      <fraction n="0.12" ref="H" />
+    </material>
+  </materials>
+
+  <!-- Solid definitions -->
+  <solids>
+    <box name="WorldSolid" x="world_size_X" y="world_size_Y" z="world_size_Z" lunit="mm"/>
```

Comment:
```suggestion
    <box name="WorldSolid" x="world_size_X + 3500 * 2" y="world_size_Y" z="world_size_Z" lunit="mm"/>
```

This fixes the overlaps, but doesn't make the world completely tight, but I think it's enough for a merge

Reply:
I removed the scintillators to check if those are the problem but they aren't.

Reply:
Hi Luis, 
thanks for your feedback and the file inspection! 
I was indeed just taking Clara's file without checking it beforehand and went for the PR. :)

I fully agree, we should have a "clean state" without any overlaps and undefined behaviours. 
I will try to have a look asap (after the CM preparations) to clean up the overlaps and minimise the world volume.

Reply:
Hi Luis, 
I've rechecked Clara's GDML file and I have to admit that Im a bit lost with respect to her implementation.
I guess that I have to dis/en-able physics-volume by physics-volume to carefully debug where the overlaps occur.
A nice geometry display would make this task significantly easier. :) 

The one you've shown above is even showing the affected overlay regions in yellow, right? 
However, this does neither seem to be the restG4 implemented geometry viewer, nor the raw root one (gGeoManager->GetTopVolume()->Draw("ogl");), nor your nice browser-implementation (https://github.com/lobis/geometry-browser.git). Could you point me to the overlap display functionality? 

I will also use this iteration to try making the GDML file a bit more readable and less dependent on different top level parameters like the world volume sizes to avoid future overlaps.

Reply:
> Hi Luis, I've rechecked Clara's GDML file and I have to admit that Im a bit lost with respect to her implementation. I guess that I have to dis/en-able physics-volume by physics-volume to carefully debug where the overlaps occur. A nice geometry display would make this task significantly easier. :)

For me the easiest way to check overlaps is to load the geometry in root (no REST-for-Physics involved) as described here: https://github.com/iaxo/iaxo-geometry/pull/9#issuecomment-2405441149, removing volumes in the gdml one by one until you find the overlap(s).

The drawings shown in this PR are from the FreeCAD GDML plug-in (https://github.com/KeithSloan/GDML). It's pretty useful, at least for visualization. To me is the quickest way to view a gdml file.

> The one you've shown above is even showing the affected overlay regions in yellow, right? However, this does neither seem to be the restG4 implemented geometry viewer, nor the raw root one (gGeoManager->GetTopVolume()->Draw("ogl");), nor your nice browser-implementation (https://github.com/lobis/geometry-browser.git). Could you point me to the overlap display functionality?

Wow, I had forgotten about that viewer! It's quite old but I'm glad it's somewhat useful!

The picture is from the FreeCAD plug-in, it paints different materials in different colors (I think) that yellow colors does not indicate overlaps, sadly. It's hard to find overlaps, I don't know a quick way to do it, let me know if you find one!

> I will also use this iteration to try making the GDML file a bit more readable and less dependent on different top level parameters like the world volume sizes to avoid future overlaps.

This is definitely a good idea!

Reply:
The FreeCAD GDML plug-in works actually very well!
In combination with yours and Maria's method "debugging" will be much easier. Thank you guys!